internal/util-interface/src/main/java/xsbti/Maybe.java

```diff
@@ -14,17 +14,36 @@ protected Maybe() {}
 		return new Maybe<s>() {
 			public boolean isDefined() { return true; }
 			public s get() { return v; }
+			public int hashCode() { return 17 + (v == null ? 0 : v.hashCode()); }
+			public String toString() { return "Maybe(" + v + ")"; }
```

Comment:
Shouldn't this be `Maybe.just(" + v + ")` so `toString` output corresponds with the Scala code to create the value?